docs/VCIssuance-vs-DataProvider.md

```diff
@@ -0,0 +1,28 @@
+# VCIssuance vs DataProvider
+
+
+Certify has a plugin model to issue VCs so that implementors can extend Inji Certify to issue various VCs and and the plugin can be of two types.
+
+1. VCIssuancePlugin
+2. DataProviderPlugin
+
+While they are two types of Issuing Plugins, both types are issuing VCs by connecting to a configured datasource. The two types of Plugins enable Inji Certify to operate differ in where VC Signing happens.
+
+## How to choose to implement either one?
+
+- An integrator can choose to implement VCIssuancePlugin interface if they want to implement the VC Signing by themselves. This gives more power to the VC Plugin authors in choosing to support their own formats, signing algorithms which may or may not be supported by Certify.
```

Comment:
This has to come out little differently, we have to say if the implementor already has a VC generation system in place and wishes to use that and use the certify only for delivering the VC in OpenID4VCI way to the holder

Reply:
Changing it to...


- An integrator can choose to implement VCIssuancePlugin interface if they want to implement the VC Signing by themselves or already have a working/existing VC Generation System. This gives more power to the VC Plugin authors in choosing to support their own formats, signing algorithms which may or may not be supported by Certify and only using the Certify for delivering the VC in an OpenID4VCI compatible way to open up to a wider ecosystem of clients which are also compatible with the standard.